src/pages/MissionAndPurpose/MissionAndPurpose.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
This file can be condensed and cleaned up a bit. No need for this additional toll on memory by creating a separate function, instead lets destructure the object and pull the data we need. Check this out below: 

```
import React from 'react';

import Container from 'components/Container/Container.js'

import './MissionAndPurpose.scss';


const createMarkup = (body) => {
  return {__html: body};
}

const MissionAndPurpose = ({ data }) =>{
  if (!data) return null;

  let { section_one, section_two, section_three } = data.mission_and_purpose;

  return(
    <Container>
      <div className="about-mission-purpose">
        <h1 className="about-mission-purpose__title">{section_one.title}</h1>
        <div className="about-mission-purpose__body" dangerouslySetInnerHTML = {createMarkup(section_one.body)}/>
        <h1 className="about-mission-purpose__title">{section_two.title}</h1>
        <div className="about-mission-purpose__body" dangerouslySetInnerHTML = {createMarkup(section_two.body)}/>
        <h1 className="about-mission-purpose__title">{section_three.title}</h1>
        <div className="about-mission-purpose__body" dangerouslySetInnerHTML = {createMarkup(section_three.body)}/>
      </div>
    </Container>
  )
}
export default MissionAndPurpose;
```